models/pipeline.js

```diff
@@ -135,5 +135,5 @@ module.exports = {
      * @property indexes
      * @type {Array}
      */
-    indexes: ['scmUri']
+    indexes: [{ fields: ['createTime'] }, { fields: ['scmUri'] }]
```

Comment:
Probably don't need the `createTime` index. It's unlikely we will query for "pipelines created during this time"
I imagine we would already know the pipeline id and try to see reports for that specific pipeline.

Reply:
this can be used to query directly; let it be; I remember someone asked me this query

Reply:
I still don't see why users would need to know about all pipelines (probably not even theirs) created during certain time. And pipeline creation is a one time operation, as opposed to builds that keep getting run every day. I think we can add it later if needed.

Reply:
its not for users; its for internal